docs-source/content/samples/simple/tanzu-kubernetes-service/_index.md

```diff
@@ -0,0 +1,763 @@
+                CalculatedListenPorts: false
+    Server:
+        'admin-server':
+            ListenPort: 7001
+    ServerTemplate:
+        'cluster-1-template':
+            Cluster: 'cluster-1'
+            ListenPort: 8001
+
+appDeployments:
+    Application:
+        myapp:
+            SourcePath: 'wlsdeploy/applications/myapp-v1'
+            ModuleType: ear
+            Target: 'cluster-1'
+```
+
+
+
+The model files:
+
+- Define a WebLogic domain with:
+    - Cluster `cluster-1`
+    - Administration Server `admin-server`
+    - A `cluster-1` targeted ear application that’s located in the WDT archive ZIP file at `wlsdeploy/applications/myapp-v1`
+
+- Leverage macros to inject external values:
+    - The property file `CLUSTER_SIZE` property is referenced in the model YAML file `DynamicClusterSize` and `MaxDynamicClusterSize` fields using a PROP macro.
+    - The model file domain name is injected using a custom environment variable named `CUSTOM_DOMAIN_NAME` using an ENV macro.
+        - You set this environment variable later in this sample using an env field in its Domain.
+        - _This conveniently provides a simple way to deploy multiple differently named domains using the same model image_.
+    - The model file administrator user name and password are set using a `weblogic-credentials` secret macro reference to the WebLogic credential secret.
+        - This secret is in turn referenced using the `webLogicCredentialsSecret` field in the Domain.
+        - The `weblogic-credentials` is a reserved name that always dereferences to the owning Domain actual WebLogic credentials secret name.
+
+A Model in Image image can contain multiple properties files, archive ZIP files, and YAML files but in this sample you use just one of each. For a complete discussion of Model in Images model file naming conventions, file loading order, and macro syntax, see [Model files]({{< relref "/userguide/managing-domains/model-in-image/model-files.md" >}}) files in the Model in Image user documentation.
+Creating the image with WIT
+
+##### Creating the image with WIT
+
+At this point, you have staged all of the files needed for image `model-in-image:WLS-v1`; they include:
+
+  - `/tmp/mii-sample/model-images/weblogic-deploy.zip`
+  - `/tmp/mii-sample/model-images/model-in-image__WLS-v1/model.10.yaml`
+  - `/tmp/mii-sample/model-images/model-in-image__WLS-v1/model.10.properties`
+  - `/tmp/mii-sample/model-images/model-in-image__WLS-v1/archive.zip`
+
+If you don’t see the `weblogic-deploy.zip` file, then you missed a step in the prerequisites.
+
+Now, you use the Image Tool to create an image named `model-in-image:WLS-v1` that’s layered on a base WebLogic image. You’ve already set up this tool during the prerequisite steps.
+
+Run the following commands to create the model image and verify that it worked:
+
+If you are taking the JRF path through the sample, then remove --chown oracle:root from the imagetool.sh command below.
```

Comment:
If you are taking the JRF path through the sample, -> I don't recall you describing anything about a JRF path through the sample